bin/disco_vpc_ui.py

```diff
@@ -107,27 +138,29 @@ def proxy_peerings_command(args):
         vpc_id = None
 
     if args.list_peerings:
-        vpc_map = {vpc.id: vpc for vpc in DiscoVPC.list_vpcs()}
+        vpc_map = {vpc['id']: vpc for vpc in DiscoVPC.list_vpcs()}
         peerings = sorted(
-            DiscoVPC.list_peerings(vpc_id, include_failed=True),
-            key=lambda p: vpc_map.get(p.accepter_vpc_info.vpc_id).tags.get("Name"))
+            DiscoVPCPeerings.list_peerings(vpc_id, include_failed=True),
+            key=lambda p: vpc_map.get(p['AccepterVpcInfo']['VpcId'])['tags'].get("Name"))
```

Comment:
I bumped to this one before a lot. i suspect you disable the pep8 to check this one.

http://stackoverflow.com/questions/25010167/e731-do-not-assign-a-lambda-expression-use-a-def